test/parallel/test-repl.js

```diff
@@ -599,7 +599,9 @@ const errorTests = [
       /^Require stack:/,
       /^- <repl>/,
       /^ {4}at .*/, // at Module._resolveFilename
-      /^ {4}at .*/, // at Module._load
+      /^ {4}at .*/, // At defaultResolveImpl
+      /^ {4}at .*/, // At resolveForCJSWithHooks
+      /^ {4}at .*/, // at Function._load
```

Comment:
Or we can update the `send` line a few lines up to be like `send: 'Error.stackTraceLimit = 1; require("internal/repl")',` and then just delete all the lines after 601 and then we don’t need to keep updating this test as the call stack within the CommonJS loader changes.